Makefile

```diff
@@ -167,16 +167,16 @@ debug-shared: shared
 static: lib/libsass.a
 shared: $(LIBRARIES)
 
-lib/libsass.a: $(COBJECTS) $(OBJECTS)
+lib:
 	$(MKDIR) lib
```

Comment:
You might be able to use the `@` prefix which will suppress any errors caused by `@mkdir ..` and continue on as normal. Not sure if that works on windows.